### PR TITLE
[CI] Run Frontend and Drivers tests on Ubuntu 22.04

### PR DESCRIPTION
**Pull request by nemanjaglumac**
We have to start running all our tests on the latest ubuntu images (22.04)

This PR first moves Frontend and Drivers tests for two reasons:
- custom `buildjet-2vcpu-ubuntu-2004` have been timing out a lot and I hope that the latest ubuntu images might have less network issues
- "re-run" workflow started failing every time it needs to re-run Driver of Frontend workflow
    - this was pointing to the [differences in custom BuildJet runners vs hosted GitHub runners](https://buildjet.com/for-github-actions/docs/runners#known-differences-between-buildjet-and-github-actions)

### CODE REVIEW

**Comment by ariya** [comment]
Is this failure due to the memory limitation of GitHub runner?

image


See https://github.com/metabase/metabase/actions/runs/4257758142/jobs/7408652399

**Comment by nemanjaglumac** [comment]
> Is this failure due to the memory limitation of GitHub runner?
> 
> image
> 


I am still investigating.

File: .github/workflows/drivers.yml [hunk 1]
```diff
@@ -563,8 +563,9 @@ jobs:
       with:
         junit-name: 'be-tests-presto-jdbc-ee'
         test-args: ":exclude-tags '[:mb/once]'"
-    - name: Capture max memory usage
-      run: cat /sys/fs/cgroup/memory/memory.max_usage_in_bytes
+    # FIXME: Cannot find it in Ubuntu 22.04 (No such file or directory)
+    # - name: Capture max memory usage
+    #   run: cat /sys/fs/cgroup/memory/memory.max_usage_in_bytes
```
Comment (nemanjaglumac) [inline .github/workflows/drivers.yml:568]: https://github.com/metabase/metabase/issues/28602

**Review by WiNloSt — APPROVED** [review]
Thanks for doing this.

**Pull request merged**